README.md

```diff
@@ -61,6 +61,35 @@ Run a single test by adding the path to the file name:
 npx wc-e2e test:e2e-dev tests/e2e/specs/activate-and-setup/complete-onboarding-wizard.test.ts
 ```
 
+### Documentation
+
+There is documentation in 2 forms available in the repo. A static set of documentation supported by docsify and also a Storybook containing component documentation for `@woocommerce/components`.
+
+To view the docsify docs locally you can do:
+
+```
+npm install
+cd docs
+npx docsify serve
+```
+
+When deployed the docsify docs also host an embedded version of the storybook docs. To generate that and test it locally in docsify you'll need to run:
+
+```
+npm install
+npm run docs
+npx docsify serve
```

Comment:
Just add a `cd docs` above `docsify serve` similar as above.

Reply:
good point thanks for this suggestion!